extensions/wikia/ArticleComments/classes/ArticleCommentsAjax.class.php

```diff
@@ -83,49 +87,52 @@ static public function axEdit() {
 		$articleId = $wgRequest->getVal( 'article', false );
 		$commentId = $wgRequest->getVal( 'id', false );
 
-		$result = array(
+		$result = [
 			'error'	=> 1,
 			'id'	=> $commentId,
 			'show'	=> false,
 			'text'	=> ''
-		);
+		];
 
-		/**
-		 * check owner of article
-		 */
+		// Check owner of article
 		$title = Title::newFromID( $articleId );
 		if ( !$title ) {
 			return $result;
 		}
 
-		/**
-		 * edit comment
-		 */
+		// Edit comment
 		$comment = ArticleComment::newFromId( $commentId );
-		if ( $comment ) {
-			$comment->load( true );
-			if ( $comment->canEdit() ) {
-				$result['error'] = 0;
-				$result['show'] = true;
-				$result['text'] = $comment->editPage();
-
-				if ( ArticleComment::isMiniEditorEnabled() ) {
-					$result['edgeCases'] = MiniEditorHelper::getEdgeCases();
-				}
-
-				$result['emptyMsg'] = wfMsg( 'article-comments-empty-comment', $comment->getTitle()->getLocalUrl( 'redirect=no&action=delete' ) );
-			}
+		if ( empty( $comment ) ) {
+			return $result;
+		}
+
+		if ( !$comment->load( true ) ) {
+			return $result;
+		}
+
+		if ( !$comment->canEdit() ) {
+			return $result;
 		}
 
+		$result['error'] = 0;
+		$result['show'] = true;
+		$result['text'] = $comment->editPage();
+
+		if ( ArticleComment::isMiniEditorEnabled() ) {
+			$result['edgeCases'] = MiniEditorHelper::getEdgeCases();
+		}
+
+		$result['emptyMsg'] = wfMessage(
+			'article-comments-empty-comment',
+			$comment->getTitle()->getLocalUrl( 'redirect=no&action=delete' )
+		)->text();
```

Comment:
Ohh. It's not so easy because we have raw HTML in the message. If you think it's not in scope of this ticket please create new one to address this issue.